_docs/tasks/security/health-check.md

```diff
@@ -140,4 +140,4 @@ continuously failed health checks.
 
 ## What's next
 
-* Read the [Istio Citadel arguments](https://github.com/istio/istio/blob/master/security/cmd/istio_ca/main.go).
+* Read the [Citadel arguments](https://github.com/istio/istio/blob/master/security/cmd/istio_ca/main.go).
```

Comment:
Please point to the reference page on istio.io instead of to the source code!
We need to rename istio_ca to citadel.
"Read the Citadel arguments" -> Read more about Citadel

Reply:
Done.